code/__HELPERS/mobs.dm

```diff
@@ -317,4 +317,11 @@ Proc for attack log creation, because really why not
 					. = 0
 					break mainloop
 	if(progbar)
-		qdel(progbar)
\ No newline at end of file
+		qdel(progbar)
+
+/proc/isabductor(A)
+	. = FALSE
+	if(ishuman(A))
+		var/mob/living/carbon/human/H = A
+		if(H.dna && istype(H.dna.species, /datum/species/abductor))
+			. = TRUE

```

Comment:
well I wanted to make a macro like the ones in code/__DEFINES/is_helpers.dm but since it requires some variable assignment to get the .dna I couldn't.

Would putting it in atom empty and then putting this body in /mob/living/carbon/human.dm be better? I thought we were trying to avoid empty procs in atom...